app/src/main/java/com/infomaniak/swisstransfer/ui/components/SwissTransferTextField.kt

```diff
@@ -89,6 +93,8 @@ fun SwissTransferTextField(
 
     OutlinedTextField(
         modifier = modifier,
+        readOnly = isReadOnly,
+        enabled = !isReadOnly,
```

Comment:
Is it mandatory to have `readOnly` and `enabled` ? Since `enabled` is the opposite of `readOnly`.

Reply:
Yes.

If we don't add `enabled`, the field is selectable. By clicking on it, it changes its border color.
We don't want this behavior.